### DISCUSSION TITLE
Which parameters matter for capacity loss

### DISCUSSION THREAD

**Discussion by skywo1f** [pybamm-team/PyBaMM#2536] [General]
I am trying to use PyBaMM to study how NMC and LFP batteries differ. One of my tests is to figure out how the capacity decays over time/cycles. I was wondering if anyone knew of any papers/guides which might tell me which parameters are important for capacity tests.

**Comment by manjunathnilugal** [comment]
Hi, 
Kindly refer to the link mentioned below for cycling and time ageing

https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/simulating-long-experiments.ipynb
https://github.com/pybamm-team/PyBaMM/blob/develop/examples/scripts/calendar_ageing.py

**Reply by skywo1f** [reply]
sure, that tells me how to simulate the system, but it doesnt mean anything if I dont have the right parameters set up in my mohtat or prada files.

**Comment by valentinsulzer** [comment]
See https://pubs.rsc.org/en/content/articlehtml/2021/cp/d1cp00359c and https://pubs.rsc.org/en/content/articlehtml/2022/cp/d2cp00417h

**Reply by Niko191292** [reply]
Hello, I am currently trying to reproduce the results of the second paper. Unfortunately, I do not manage to add Li plating and particle cracking. Is there any code available for this ?
